### ISSUE TITLE
WCAG 2.1 1.4.12 Text spacing in the Switcher 

### ISSUE THREAD

**Issue by ferBonnin** [microsoft/accessibility-insights-web#1219]
## Describe the bug

The switcher to change from fastpass to assessment has overlapping text

**To Reproduce**
1. Go to  text legibility> text spacing
2. run the following function (Same as the bookmarklet)
`(function () { document.querySelector("body").classList.add("text-spacing"); var style = document.createElement("style"), styleContent = document.createTextNode(".text-spacing * { line-height: 1.5 !important; letter-spacing: 0.12em !important; word-spacing: 0.16em !important; } .text-spacing p{ margin-bottom: 2em !important; } "); style.appendChild(styleContent); var caput = document.getElementsByTagName("head"); caput[0].appendChild(style); })();
`
3. check the switcher

## Expected behavior

all text is visible

## Screenshots

![Switcher with overlapping text](https://user-images.githubusercontent.com/9062104/64457949-ecf3e200-d0a8-11e9-9ed5-e999d033393a.png)

## Extension (please complete the following information)

- OS: [e.g. Windows/Mac] Windows
- Version [e.g. 1.140.1] 2.9.1
- Browser Version [e.g Chrome 71.0] 75
- Target Page [e.g www.bing.com] any

## Are you willing to submit a PR?

No
## Did you search for similar existing issues?

Yes

**Comment by AhmedAbdoOrtiga** [comment]
I can't seem to repro, the only difference is my Chrome version is "Version 76.0.3809.132 (Official Build) (64-bit)", can you please advise?

**Comment by AhmedAbdoOrtiga** [comment]
Worked with @peterdur, we were able to repro it by running the code in the DevTool console

**Comment by peterdur** [comment]
Looks to be related to .ms-Dropdown-title having width: 139px;

**Comment by varshannagarajan** [comment]
Hey, I was wondering if I could work on this issue.

**Comment by ferBonnin** [comment]
Hi @varshannagarajan for sure! thanks for the help :)

**Comment by varshannagarajan** [comment]
Hi, @ferBonnin I was trying out a few different changes and was wondering if the drop down box should remain the same size or if it was OK if it changed according to the length of the text?

Static:
![StaticDropdown](https://user-images.githubusercontent.com/48139461/67823211-8e3f5900-fa98-11e9-836a-c686affa117d.gif)

Dynamic:
![DynamicDropdown](https://user-images.githubusercontent.com/48139461/67823223-9a2b1b00-fa98-11e9-949c-19f2b7d69a76.gif)

**Comment by ferBonnin** [comment]
hi @varshannagarajan thanks for adding the gif and experimenting with different ideas :) 
I think it would be better to have the dropdown box size static. That part of the UI doesn't really change much for the users and it feels more consistent if the size doesn't change. 

I noticed from the gif that the dropdown is a bit bigger than the title bar, is it just the gif? if not, could you please make sure the dropdown stays contained in the title bar?
![screenshot of the ddl, it looks slightly overflow](https://user-images.githubusercontent.com/9062104/67881346-97164600-fafd-11e9-8bfe-8fe876955007.png)

thanks again for the awesome contribution!

**Comment by varshannagarajan** [comment]
Hi , I have submitted a pull request to just address the text overlap. I tried a couple of different things to make sure the drop down box would not go below the header (when the function is applied) but wasn't yet able to find a good solution. I was wondering if it would be OK to change the height of the header or if that must stay the same?

I submitted the PR now just to get it in before the end of Hacktoberfest, I hope that is alright. I will continue to work on the shifting issue and I will submit a PR for it. 

Also I did notice when the function is applied on the side menu the text is cut off.

![image](https://user-images.githubusercontent.com/48139461/68057890-ba8feb00-fccd-11e9-8139-1667551ee09f.png)

I believe is would also be a problem with WCAG 2.1 1.4.12 and was wondering if you would like me to address that as well.

And no problem I am happy to help!

**Comment by ferBonnin** [comment]
Hi @varshannagarajan thanks for the PR! Since you plan on keep working on this, I have marked your PR as do not merge for now until we are ready, sounds good?

I am adding @peterdur to the thread, can we help with varshannagarajan to fix the drop down box without changing the size of the header?

for the side menu, great point! if you want to file an issue and help us fix it that would be great :)

**Comment by ferBonnin** [comment]
Marking this as resolved :)

**Comment by varshannagarajan** [comment]
Hi, I was wondering if you would like me to create a new separate issue regarding the drop down box moving outside the header?

**Comment by ferBonnin** [comment]
validated this issue in canary and insider, looks good :)

**Comment by ferBonnin** [comment]
> Hi, I was wondering if you would like me to create a new separate issue regarding the drop down box moving outside the header?

yes, please!
